packages/renderer/src/lib/cronjob/CronJobColumnActions.svelte

```diff
@@ -0,0 +1,7 @@
+<script lang="ts">
+import CronJobActions from './CronJobActions.svelte';
+
+let { object } = $props();
```

Comment:
this not typed ?

You can look at https://github.com/podman-desktop/podman-desktop/pull/10906#discussion_r1935833173 for some details to use Props